site/docs/v0.20.0/README.md

```diff
@@ -0,0 +1,204 @@
+# <img src="img/sonobuoy-logo.png" width="400px" alt="Sonobuoy logo" > [![CircleCI](https://circleci.com/gh/vmware-tanzu/sonobuoy.svg?style=svg)](https://circleci.com/gh/vmware-tanzu/sonobuoy)
+
+## [Overview][oview]
+
+Sonobuoy is a diagnostic tool that makes it easier to understand the
+state of a Kubernetes cluster by running a set of plugins (including [Kubernetes][k8s] conformance
+tests) in an accessible and non-destructive manner. It is a customizable,
+extendable, and cluster-agnostic way to generate clear, informative reports
+about your cluster.
+
+Its selective data dumps of Kubernetes resource objects and cluster nodes allow
+for the following use cases:
+
+* Integrated end-to-end (e2e) [conformance-testing][e2ePlugin]
+* Workload debugging
+* Custom data collection via extensible plugins
+
+Starting v0.20, Sonobuoy supports Kubernetes v1.17 or later.
+Sonobuoy no longer follows Kubernetes release cycle -- this means that Sonobuoy releases in the future may not follow the same cadence as Kubernetes releases.
```

Comment:
Makes it sound affirmative that Sonobuoy will not follow kubernetes release cadance.